package.json

```diff
@@ -17,14 +17,15 @@
     "compile": "tsc ./tsconfig.build.json --resolveJsonModule",
     "lint": "eslint --fix src",
     "style": "prettier --write ./src/**/*.ts",
-    "test": "mocha -r ts-node/register 'test/**/*.test.ts' --exit --config test/.mocharc.json --timeout 120000",
+    "test": "mocha -r ts-node/register 'test/**/e2e.test.ts' --exit --config test/.mocharc.json --timeout 150000",
```

Comment:
only e2e tests? You can run `yarn test` and usually specific a specific test file to run if you just want to test a single file